plugins/inputs/nfsclient/README.md

```diff
@@ -0,0 +1,136 @@
+        - vfswritepages - (int, count) - Count of how many times a _group_ of pages was written (possibly via `mmap()`?)
+        - vfsgetdents - (int, count) - Count of directory entry reads with getdents(). These reads can be served from cache and don't necessarily imply actual NFS requests. (formerly called "vfsreaddir")
+        - vfssetattr - (int, count) - How many times we've set attributes on inodes.
+        - vfsflush - (int, count) - Count of times pending writes have been forcibly flushed to the server.
+        - vfsfsync - (int, count) - Count of calls to `fsync()` on directories and files.
+        - vfslock - (int, count) - Number of times a lock was attempted on a file (regardless of success or not).
+        - vfsrelease - (int, count) - Number of calls to `close()`.
+        - congestionwait - (int, count) - Unused.
+        - setattrtrunc - (int, count) - How many times files have had their size truncated.
+        - extendwrite - (int, count) - How many times a file has been grown because you're writing beyond the existing end of the file.
+        - sillyrenames - (int, count) - Number of times an in-use file was removed (thus creating a temporary ".nfsXXXXXX" file)
+        - shortreads - (int, count) - Number of times the NFS server returned less data than requested.
+        - shortwrites - (int, count) - Number of times NFS server reports it wrote less data than requested.
+        - delay - (int, count) - Occurances of EJUKEBOX ("Jukebox Delay", probably unused)
+        - pnfsreads - (int, count) - Count of NFS v4.1+ pNFS reads.
+        - pnfswrites - (int, count) - Count of NFS v4.1+ pNFS writes.
+
+  - nfs_xprt_tcp
+    - tags:
+        - mountpoint - The local mountpoint, for instance: "/var/www"
+        - serverexport - The full server export, for instance: "nfsserver.example.org:/export"
+    - fields:
+        - bind_count - (int, count) - Number of _completely new_ mounts to this server (sometimes 0?)
+        - connect_count - (int, count) - How many times the client has connected to the server in question
+        - connect_time - (int, jiffies) - How long the NFS client has spent waiting for its connection(s) to the server to be established.
+        - idle_time - (int, seconds) - How long (in seconds) since the NFS mount saw any RPC traffic.
+        - rpcsends - (int, count) - How many RPC requests this mount has sent to the server.
+        - rpcreceives - (int, count) - How many RPC replies this mount has received from the server.
+        - badxids - (int, count) - Count of XIDs sent by the server that the client doesn't know about.
+        - inflightsends - (int, count) - Number of outstanding requests; always >1.  ("Every time we send a request, we add the current difference between sends and receives to this number. Since we've just sent a request, this is always going to be at least one. This number is not as useful as you think it should be.")
```

Comment:
Please remove the comment in brackets.

Reply:
The comment is quoted from  from https://utcc.utoronto.ca/~cks/space/blog/linux/NFSMountstatsXprt
Removed comment, and pointed to reference for more information.

Reply:
Please mark as resolved.